example.rules

```diff
@@ -0,0 +1,9 @@
+# The count of CPUs per node, useful for getting CPU time as a percent of total.
+instance:node_cpu:count = count(node_cpu{mode="idle"}) without (cpu,mode)
```

Comment:
I see what the rule is doing, and the name of the metric indicates it's doing something else. You need to give this metric a different name.